odl/discr/lp_discr.py

```diff
@@ -427,6 +511,73 @@ def cell_volume(self):
         """Cell volume of an underlying regular grid."""
         return self.space.cell_volume
 
+    @property
+    def real(self):
+        """Real part of this element."""
+        return self.space.as_real_space().element(self.asarray().real)
+
+    @real.setter
```

Comment:
This mimics Numpy's behavior to make `real` and `imag` writable properties. Note that `x.real` may create a copy so you need a setter to make it effectively writable.